mm2src/coins/utxo/utxo_standard.rs

```diff
@@ -163,7 +163,12 @@ impl UtxoCommonOps for UtxoStandardCoin {
     }
 
     async fn get_current_mtp(&self) -> UtxoRpcResult<u32> {
-        utxo_common::get_current_mtp(&self.utxo_arc, CoinVariant::Standard).await
+        let coin_variant = if self.ticker() == "LBC" {
```

Comment:
Could you please add `coin_variant_by_ticker` helper instead and use it everywhere?